eks-anywhere-common/Testers/newrelic/metadata-injection-test-job.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: newrelic-testjob
+  namespace: newrelic
+spec:
+  backoffLimit: 1
+  template:
+    spec:
+      initContainers:
+        - name: busybox
+          image: busybox
+          command: ['sh', '-c', "until nslookup newrelic-bundle-nri-metadata-injection.newrelic.svc.cluster.local; do echo waiting for myservice; sleep 10; done"]
```

Comment:
@anshrma Change the command to `until nslookup newrelic-newrelic-nri-metadata-injection.newrelic.svc.cluster.local; do echo waiting for myservice; sleep 10; done`